src/Chapter1.hs

```diff
@@ -520,8 +520,8 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+closestToZero x y = if abs x < abs y then x else y
+--closestToZero x y = min (abs x) (abs y)
```

Comment:
Unfortunately, this solution will return the absolute value instead of the value itself.
So the above one is the way to go 😉